packages/core/src/package.ts

```diff
@@ -306,18 +306,24 @@ export class Package {
     const localDependencies = this.retrievePackageDependencies(depName);
     const updatingDependencies = [localDependencies];
 
-    // when we have peer dependencies, we might need to perform certain things
+    // when we have peer dependencies, we might need to perform certain actions
     if (this.peerDependencies?.[depName]) {
       // when user allows peer bump and is a regular semver version, we'll push it to the array of dependencies to potentially bump
-      // however we won't when the semver has a range with operator, ie this would bump ("^2.0.0") but these would not (">=2.0.0" or "workspace:<2.0.0" or "workspace:*")
+      // however we won't when the semver has a range with operator, ie this would bump ("^2.0.0") but the following would not (">=2.0.0" or "workspace:<2.0.0" or "workspace:*")
```

Comment:
actually re-reading the comments I left and if we now support `workspace:^` (and `~`) then we might as well support `workspace:*` too (which would equal to current version)

Reply:
done